gdp-src-build/conf/templates/silk.bblayers.conf

```diff
@@ -0,0 +1,7 @@
+include templates/bblayers.inc
+
+# porter specific layer configuration
+BBLAYERS += " \
+  ${TOPDIR}/../meta-renesas \
+  ${TOPDIR}/../meta-renesas/meta-rcar-gen2 \
```

Comment:
@slawr I thought that might be the issue, but I can successfully build porter from the single branch without it complaining about missing the multimedia else I would have caught this sooner. Your explanation makes sense to me.